git/foundations_git/git_basics.md

```diff
@@ -94,7 +94,7 @@ Finally, let's upload your work to the GitHub repository you created at the star
 
 1. <span id="git-push"></span>Type `git push`. To be more specific, type `git push origin main`. Since you are not dealing with another branch (other than *main*) or a different remote (as mentioned above), you can leave it as `git push` to save a few keystrokes. **NOTE: If at this point you receive a message that says "Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.", you have followed the steps incorrectly and cloned with HTTPS, not SSH. Please follow the steps for [switching remote URLs from HTTPS to SSH](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#switching-remote-urls-from-https-to-ssh) to change your remote to SSH, then attempt to push to Github.**
 
-1. Type `git status` one final time. It should output "*Your branch is up to date with 'origin/main'. nothing to commit, working tree clean*".
+1. Type `git status` one final time. It should output "*Your branch is up to date with 'origin/main'. Nothing to commit, working tree clean*".
```

Comment:
```suggestion
1. Type `git status` one final time. It should output "*Your branch is up to date with 'origin/main'. nothing to commit, working tree clean*".
```
This is a demonstration of what the command `git status` outputs in the terminal when there are no changes to report. Notice that "nothing" isn't capitalized.
![Selection_428](https://github.com/TheOdinProject/curriculum/assets/88392688/c9355f59-e688-4b9d-af42-dc93dd3cd29d)